.github/workflows/test-rust.yaml

```
@@ -57,37 +70,88 @@ jobs:
+            cmake --build . --config Release -- -m:2
+            cmake --build . --config Release --target install -- -m:2
+          else
+            make -j2
+            make install
+          fi
           ls -lh install/lib
 
-          echo "SHERPA_ONNX_LIB_DIR=$PWD/install/lib" >> "$GITHUB_ENV"
+      - name: Set library paths
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2193:warning:4:28: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)